Sources/Fluent/Query/Builder/QueryBuilder.swift

```diff
@@ -5,26 +5,26 @@ import Foundation
 public final class QueryBuilder<Model, Result> where Model: Fluent.Model, Model.Database: QuerySupporting {
     /// The query being built.
     public var query: DatabaseQuery<Model.Database>
-
+    
```

Comment:
there's an xcode option to delete whitespace on empty lines. please enable that to reduce the git noise here :)